config/katello.migrations/171030135054-foreman-proxy-update.rb

```diff
@@ -0,0 +1,5 @@
+# foreman_proxy defaults
+answers['foreman_proxy'] = {
+  'http' => true,
+  'http_port' => '8000'
```

Comment:
These would also need to be configured in the scenario answers file for fresh install to get them.